src/components/PulsatingEffect/PulsatingEffect.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from "react";
+
+export function PulsatingEffect({
+  targetSelector,
+}: {
+  targetSelector: string;
+}): JSX.Element {
+  useEffect(() => {
+    const element = document.querySelector<HTMLElement>(targetSelector);
+    if (element) {
+      element.style.animation = "pulsate 3s infinite";
+      element.style.transformOrigin = "center";
+      element.style.animationTimingFunction = "ease-in-out";
+
+      const keyframes = `
+        @keyframes pulsate {
+          0% {
+            box-shadow: 0 0 0 0 rgba(2, 156, 253, 0.7), 0 0 0 0 rgba(2, 156, 253, 0.4);
+          }
+          50% {
+            box-shadow: 0 0 0 20px rgba(2, 156, 253, 0), 0 0 0 40px rgba(2, 156, 253, 0);
+          }
+          100% {
+            box-shadow: 0 0 0 0 rgba(2, 156, 253, 0), 0 0 0 0 rgba(2, 156, 253, 0);
+          }
+        }
+      `;
+      const style = document.createElement("style");
+      style.id = "sb-onboarding-pulsating-effect";
+      style.innerHTML = keyframes;
+      document.head.appendChild(style);
```

Comment:
we should remove the style from the head as well when the component gets unmounted

Reply:
I'm not sure what you mean, the logic to remove it is a couple lines below?

Reply:
Yeah, of course.